Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.xaml

```diff
@@ -10,6 +10,8 @@
         <Setter Property="ActivationWidth" Value="80" />
         <Setter Property="IsOffsetLimited" Value="True" />
         <Setter Property="ExtraSwipeThreshold" Value="40" />
+        <Setter Property="IsTabStop" Value="False" />
+        <Setter Property="TabIndex" Value="1" />
```

Comment:
Good question, I think I have tried it before to add the "IsTabStop" and forgotten to delete it.